File: service/scaladsl/integration-tests/src/test/scala/com/lightbend/lagom/scaladsl/it/routers/AdditionalRoutersService.scala
/*
 * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
 */

package com.lightbend.lagom.scaladsl.it.routers

import com.lightbend.lagom.scaladsl.api.Service.named
import com.lightbend.lagom.scaladsl.api.Descriptor
import com.lightbend.lagom.scaladsl.api.Service

trait AdditionalRoutersService extends Service {
  override def descriptor: Descriptor = named("additional-routers")
}

File: service/javadsl/integration-tests/src/test/java/com/lightbend/lagom/it/routers/AdditionalRoutersService.java
/*
 * Copyright (C) Lightbend Inc. <https://www.lightbend.com>
 */

package com.lightbend.lagom.it.routers;

import com.lightbend.lagom.javadsl.api.Descriptor;
import com.lightbend.lagom.javadsl.api.Service;

import akka.NotUsed;
import com.lightbend.lagom.javadsl.api.*;
import static com.lightbend.lagom.javadsl.api.Service.*;

public interface AdditionalRoutersService extends Service {

  @Override
  default Descriptor descriptor() {
    return named("additional-routers");
  }
}
